WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewModel.swift

```diff
@@ -19,7 +19,8 @@ final class ShippingLabelAddressFormViewModel: NSObject {
     let type: ShipType
     private(set) var address: ShippingLabelAddress?
     private(set) var addressValidationError: ShippingLabelAddressValidationError?
-    private(set) var isAddressValidated: Bool = false
+    private(set) var isAddressLocallyValidated: Bool = false
+    private(set) var isAddressFullyValidated: Bool = false
```

Comment:
Are these meant to be used later? I only see them being used in tests?
In case you do need them, maybe an enum could help you model this better.

```swift
enum Validation {
  case none
  case local
  case remote
}
```